### DISCUSSION TITLE
unable copy roboshop.conf file in ansible

### DISCUSSION THREAD

**Discussion by naveen2513** [learndevopsonline/learndevopsonline#999] [Q&A] [answered]
hi @r-devops 
workstation ip: 44.205.10.2
jenkins:34.201.17.6
![2023-06-02 (3)](https://github.com/learndevopsonline/learndevopsonline/assets/129704698/407fa63c-4639-4612-b5f5-997e099b027e)
![2023-06-02](https://github.com/learndevopsonline/learndevopsonline/assets/129704698/957cff1c-bf28-4d52-a33b-fd2c306694a0)

**Comment by learninguser** [comment, marked as answer]
Hi,

You have given an incorrect name for templates folder inside frontend role. It should be **templates** but not template. Kindly correct the same and re-run the Jenkins Job. Please revert back if you still face the issue.

Thank you.
Kind regards
Pavan Kumar.
